styles/components/image.css

```diff
@@ -0,0 +1,26 @@
+.img-res {
+  width: 100%;
+  height: 100%;
```

Comment:
style suggested to make responsive : width: 100%; height: auto;

Reply:
Thanks for reviewing Himanshu. I tried keeping it as auto but then it overflows its container in my horizontal card. So had to keep it as 100%